src/Sonarr.Http/Authentication/AuthenticationBuilderExtensions.cs

```diff
@@ -10,6 +11,8 @@ namespace Sonarr.Http.Authentication
 {
     public static class AuthenticationBuilderExtensions
     {
+        private static readonly Regex CookieNameRegex = new Regex(@"[^a-z0-9]", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
Since there is nothing else to match the `+` doesn't do anything more.

Given this code:
```
var singleRegex = new Regex(@"[^a-z0-9]", RegexOptions.Compiled | RegexOptions.IgnoreCase);
var multiRegex = new Regex(@"[^a-z0-9]", RegexOptions.Compiled | RegexOptions.IgnoreCase);

var input = "Sonarr (Test)!@#$%^&*()_+-=[]{}";
```

For both regexes the result is the same, `SonarrTest`

Reply:
Yeah, at the scale we're doing it (once), it won't make a realistic difference, but at 1 million iterations it's about 4x more efficient; regardless a good suggestion.